shared/haml/course_explorer_table.haml

```diff
@@ -53,8 +82,8 @@
     starts: 9,
     ends: 12,
     range: "Grades 9-12",
-    regular_order: 0,
-    responsive_small_order: 2,
```

Comment:
we also updated the ordering of courses in the responsive view, which is why some of these values have been updated.